utils/cli/cli.c

```diff
@@ -370,78 +370,34 @@ static int cmd_codec(int argc, char *argv[]) {
 		return EXIT_SUCCESS;
 	}
 
-	DBusMessage *msg = NULL, *rep = NULL;
-	const char *codec = argv[2];
 	int result = EXIT_FAILURE;
+	const char *codec = argv[2];
 
-	if ((msg = dbus_message_new_method_call(dbus_ctx.ba_service, path,
-					BLUEALSA_INTERFACE_PCM, "SelectCodec")) == NULL) {
-		dbus_set_error(&err, DBUS_ERROR_NO_MEMORY, NULL);
-		goto fail;
-	}
-
-	DBusMessageIter iter;
-	dbus_message_iter_init_append(msg, &iter);
-
-	if (!dbus_message_iter_append_basic(&iter, DBUS_TYPE_STRING, &codec)) {
-		dbus_set_error(&err, DBUS_ERROR_NO_MEMORY, NULL);
-		goto fail;
-	}
-
-	DBusMessageIter props;
-	if (!dbus_message_iter_open_container(&iter, DBUS_TYPE_ARRAY, "{sv}", &props)) {
-		dbus_set_error(&err, DBUS_ERROR_NO_MEMORY, NULL);
-		goto fail;
-	}
+	uint8_t data[64];
```

Comment:
Name this variable as `configuration`. Now there is no `char *property = "Configuration"` so it will be harder to guess what kind of data it is.